backend/tournesol/migrations/0058_remove_poll_moderation.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.7 on 2023-12-04 09:58
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tournesol", "0057_entitycontext_entitycontextlocale"),
```

Comment:
Another migration 0058 depending on 0057 has been merged in #1861 
So we may need to update the migration before merging this PR.